tips.json

```diff
@@ -1,52 +1,87 @@
   },
-  { 
+  {
     "tool": "Command Line",
     "toolSlug": "terminal",
     "text": "Use the [UP arrow] to cycle through previously run commands. This is useful when you're doing repetitive tasks like pushing commits to GitHub!"
   },
-  { 
+  {
     "tool": "Command Line",
     "toolSlug": "terminal",
     "text": "Use [ctrl + l] to clear the screen. This doesn't lose any of your previous commands, but can help if you need a visual break! Neat, huh?"
   },
   {
+    "tool": "Command Line",
+    "toolSlug": "terminal",
+    "text": "Use [ctrl + a] to go to the beginning of the line without losing any characters present on the command line."
+  },
+  {
+    "tool": "Command Line",
+    "toolSlug": "terminal",
+    "text": "Use '!!' to repeat your previous command. Because we love repeating ourselves right?"
+  },
+  {
+    "tool": "Command Line",
+    "toolSlug": "terminal",
+    "text": "Hold down the Option key while scrolling forward or backwards for lightning fast speeds!"
+  },
+  {
+    "tool": "Command Line",
+    "toolSlug": "terminal",
+    "text": "Use [cmd + number] to access the tab corresponding to the number. For example, to quicky switch to the third tab use [cmd + 3]."
+  },
+  {
+    "tool": "Command Line",
+    "toolSlug": "terminal",
+    "text": "Use 'history | grep <search term>' to search your history for a particular command. For example, 'history | grep npm' to find all commands you've used that match npm."
+  },
+  {
     "tool": "OS X",
     "toolSlug": "osx",
     "text": "[cmd + tab] will help you qickly cycle through your open applications instead of having to click on them from the dock. So fast!"
   },
   {
+    "tool": "OS X",
+    "toolSlug": "osx",
+    "text": "[cmd + w] will close the current tab in most applications, now you can quicky close the 14/30 tabs you don't need in Chrome."
+  },
+  {
+    "tool": "OS X",
+    "toolSlug": "osx",
+    "text": "[cmd + q] will close the current application -- maybe you don't need any of those tabs!"
+  },
+  {
     "tool": "EditorConfig",
     "toolSlug": "editors",
     "source": "http://editorconfig.org/",
     "text": "You can use the [.editorconfig] format to maintain consistent coding styles between different editors and IDEs. Excellent if you are working in a group!"
   },
   {
-    "tool": "Git",
-    "toolSlug": "git",
-    "source": "https://git-scm.com/book/en/v2/Git-Basics-Git-Aliases",
-    "text": "Create your own aliases in your [.gitconfig] file, so you don't need to type long command lines. Show others how to deploy with a simple [>git deploy]!"
-  },
-  { 
     "tool": "Sublime Linter",
     "toolSlug": "sublimetext",
     "source": "http://www.sublimelinter.com",
     "text": "Everyone makes mistakes, keep yours in sight using a proper linting plugin. Don't be the one that forgot that ';'!"
-  }
+  },
```

Comment:
Trailing commas are actually invalid in JSON files. Can you remove this little guy?

Reply:
woops sorry, habit LOl